Sources/TSCBasic/Process.swift

```diff
@@ -370,6 +379,14 @@ public final class Process: ObjectIdentifierProtocol {
         posix_spawn_file_actions_init(&fileActions)
         defer { posix_spawn_file_actions_destroy(&fileActions) }
 
+        if let workingDirectory = workingDirectory?.pathString {
+            if #available(macOS 10.15, *) {
```

Comment:
Hm, what do you think about marking the initializer with workingDirectory param with availability? That way, we can can avoid surprise runtime errors.

Reply:
good idea, less errors to handle too